### ISSUE TITLE
Determinants of maternal and child undernutrition

### ISSUE THREAD

**Issue by ernestguevarra** [OxfordIHTM/ihtm-hackathon-2024#9]

**Comment by AnitaMakori** [comment]
**Initial thoughts:** 

- Develop a regression model for determinants of child undernutrition
- Develop a regression model for determinants of maternal undernutrition
- Child: outcome variables (haz,waz,whz) - categorize them into (use WHO categorization guidelines) undernourished or normal
- Maternal: outcome variables (muac) - categorize into undernourished or normal
- Child-related determinants: 
- Maternal-related determinants:

**Comment by jillianfrancise** [comment]
CHILD:
haz, waz, whz
<-3: severely stunted/underweight/wasted
<=-3 to <=-2: stunted/underweight/wasted
<-2 to <2: normal

**Comment by jillianfrancise** [comment]
MATERNAL:
muac
<23.5 cm: underweight
>29.2 cm: obese

**Comment by Richmondapearce** [comment]
Maternal-related determinants: location, age first parity, ANC visits, any iron folic acid, vitamin A, post-natal, tetanus doses, MDDW

**Comment by ernestguevarra** [comment]
Team @OxfordIHTM/sakura, I had a meeting with @Richmondapearce today and I got a good sense of what your plans are for the model. I think the plan is good, I look forward to seeing each of your contributions!

**Comment by ernestguevarra** [comment]
team @OxfordIHTM/sakura, I have not seen activity from your side with regard to code which is not a problem. I am just checking in to see how things are going and whether you need help/assistance or if you have any further questions?

**Comment by ernestguevarra** [comment]
Hi Team @OxfordIHTM/sakura, just checking in to see how you guys are. I didn't see a pull request from your team so just wanted to see what you are planning? Will you still be working on this or would you like me to look at whatever branch you have been working on and has most or all of you new outputs.

I am happy to let you keep on working on this so that you can get your outputs onto the Rmarkdown. But don't feel you have to put in anymore extra time to this other than what you have already done. I can pick up from where you left off and then add your outputs into the report if you just want to stop it here and not continue anymore. Just let me know what you want to do.

In case you want to see how the current report looks at the moment with the inputs of the teams who have made a pull request (I am still waiting on some other team for their pull request), visit this link:

https://oxford-ihtm.io/ihtm-hackathon-2024

Well done guys!

**Comment by yihseong** [comment]
Dear Ernest,

Thanks for checking in, and I hope this reply finds you well.

We are still working on our contribution - some minor hitches we couldn't
immediately resolve yesterday - and will try our best to submit our pull
request by the end of the week.

Thank you for your understanding and consideration, and have a wonderful
week ahead!


Best regards,
Yih
